CMake/CMakeLibs.cmake

```diff
@@ -57,6 +62,9 @@ macro(ADD_OSQUERY_LINK IS_CORE LINK)
 endmacro(ADD_OSQUERY_LINK)
 
 macro(ADD_OSQUERY_LINK_INTERNAL LINK LINK_PATHS LINK_SET)
+  set(LINK_PATHS
+    ${LINK_PATHS} /usr/lib /usr/local/lib "$ENV{HOME}"
+    "${CMAKE_BUILD_DIR}/third-party/*/lib")
```

Comment:
This should be the inverse, we should prefer our locally-built libraries, which may have patches not yet upstreamed.